### PR TITLE
checking block calendar validity for copy

### PR DESCRIPTION
**Pull request by dvdn**
Copy calendar blocks in a season's clone if it's valid during season period.

I'll add test to this.

To be able to test, user need to have a season, valid data on a coverage (calendars),  a model uploaded and set for a route at least.

### CODE REVIEW

File: Services/CalendarManager.php [hunk 1]
```diff
@@ -408,4 +409,26 @@ public function getCalendarsForRoute($externalCoverageId, $externalRouteId, \Dat
...
+        $externalCoverageId = $season->getPerimeter()->getExternalCoverageId();
+        $calendarsData = $this->navitia->getCalendar($externalCoverageId, $calendarId);
+        $calendar = $calendarsData->calendars[0];
+        $calendarBeginDate = new \DateTime($calendar->active_periods[0]->begin);
+        $calendarEndDate = new \DateTime($calendar->active_periods[0]->end);
+
+        if ($season->getStartDate() < $calendarBeginDate && $calendarBeginDate < $season->getEndDate()) {
+            return true;
+        }
+        if ($season->getStartDate() < $calendarEndDate && $calendarEndDate < $season->getEndDate()) {
```
Comment (Netmisa) [inline Services/CalendarManager.php]: What do you think if we merge this conditions (line l427 & l424) by one :
```
(($season->getStartDate() < $calendarBeginDate && $calendarBeginDate < $season->getEndDate())  || ($season->getStartDate() < $calendarEndDate && $calendarEndDate < $season->getEndDate()))
```
Or maybe better, return directly this condition ?

Comment (dvdn) [inline Services/CalendarManager.php]: Ok it cleans some lines,
but not sure if multiple conditions in one row improves performance ?

Comment (Netmisa) [inline Services/CalendarManager.php]: Yes, is only for cleaning lines. But not for performance.

**Comment by dvdn** [comment]
Waiting for dedicated remote server,
To be able to test locally:
- modify .../Selenium/SeleneseBlockTest.php
array(
            'name' => 'your browserName',
            'browser' => '*browserName path to browser',
            'host' => 'localhost',
        )
- launch your selenium driver

**Comment by Netmisa** [comment]
Work for me

**Pull request merged**